Applications/Games/Subnautica/Steam/script.js

```diff
@@ -15,21 +14,23 @@ var installerImplementation = {
             .author("Zemogiter")
             .applicationHomepage("https://unknownworlds.com/subnautica/")
             .wineDistribution("upstream")
-            .wineVersion("4.0-rc1")
+            .wineVersion(LATEST_STABLE_VERSION)
             .wineArchitecture("amd64")
             .appId(264710)
             .preInstall(function (wine, wizard) {
+                wizard.message(tr("Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else this game might not work correctly."));
```

Comment:
I wonder if this should simply be added to the dxvk verb.

Reply:
Makes sense.

Reply:
Then maybe change it to be an actual message.

Reply:
Those print messages dosen't show up in script execution.